package.json

```diff
@@ -53,6 +53,8 @@
   "dependencies": {
     "babel-runtime": "^6.5.0",
     "deep-equal": "^1.0.1",
+    "moment": "^2.15.0",
```

Comment:
moment is a peer dependency of the react component I'm using. How should I handle that?